lnwallet/channel.go

```diff
@@ -5875,7 +5947,7 @@ func (lc *LightningChannel) validateAddHtlc(pd *PaymentDescriptor) error {
 	// possible for us to add the HTLC.
 	err = lc.validateCommitmentSanity(
 		lc.remoteUpdateLog.logIndex, lc.localUpdateLog.logIndex,
-		false, pd, nil,
+		false, false, pd, nil,
```

Comment:
rationale for `false` here?

Reply:
my bad, that was a mistake but would have been caught by the tests.